### ISSUE TITLE
Library incompatibility

### ISSUE THREAD

**Issue by mehdipaco** [spomjaksilp/pyMeCom#6]
I was having problem running the example code using the crc library in the project on windows 7. After contacting the company received the link to the [compatible PyCRC library](https://github.com/alexbutirskiy/PyCRC) and everything works fine. Hope it solves your problem.

**Comment by tt-aqt** [comment]
LOL I was just having the same problem.  It worked in Anaconda but only because I did not use 0.92 from the source suggested on the front page of this repository.

**Comment by jscman** [comment]
The front page library uses different command set, if someone wants to use the pycrc lib uncomment the import of PyCRC.CRCCCITT and add the following to the head of mecom.py:
```Python
# from PyCRC.CRCCCITT import CRCCCITT
import pycrc.algorithms
class CRCCCITT:
    def __init__(self):
        self.crc_model = pycrc.algorithms.Crc(width = 16, poly = 0x1021,reflect_in = False,
                    xor_in = 0x0000,reflect_out = False, xor_out = 0x0000)

    def calculate(self, input_data=None):
        return self.crc_model.bit_by_bit_fast(input_data)
```

**Comment by spomjaksilp** [comment]
@jscman are you willing to provide a PR?

**Comment by marc-haberstroh** [comment]
Please note that the required package is 'pythoncrc' on PyPI, not 'pycrc'.

**Comment by tt-aqt** [comment]
> 
> 
> Please note that the required package is 'pythoncrc' on PyPI, not 'pycrc'.

when i do pip3 install pythoncrc it installs, but then the import statement in mecom.py fails?  doesn't this have to be changed as well?

from PyCRC.CRCCCITT import CRCCCITT
ModuleNotFoundError: No module named 'PyCRC.CRCCCITT'

**Comment by marc-haberstroh** [comment]
No change of the import statement is needed.

I couldn't reproduce the issue you are seeing.
Please see the following.


```
habersmn@habersmn ~ % pip list
Package    Version
---------- -------
pip        21.0.1
pythoncrc  1.21
setuptools 41.2.0
habersmn@habersmn ~ % python
Python 3.8.2 (default, Dec 30 2020, 12:55:23) 
[Clang 12.0.0 (clang-1200.0.32.28)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from PyCRC.CRCCCITT import CRCCCITT
>>> 

```

**Comment by timhellwig** [comment]
Fixed in 2d089025036730b8605629fa093653bf4401c647
